templates/docs.html

```diff
@@ -77,6 +77,7 @@
 	</ul>
 	{% endif %}
 </li>
+<li class="navleft-item"><a href="/docs/tools/">Tools</a></li>
```

Comment:
I am not sure if this still applies since all other URLs are not like this and we manage quite a lot of the things with absolute URLs and redirects ....